docs/source/streamflow-public-testnet.md

```diff
@@ -0,0 +1,312 @@
+
+```
+13. Invoke "unlock broadcasting funds"
+```
+
+Upon selecting the option, you should be prompted to confirm that you would like to request to unlock your funds. After answering the wizard's prompt, you should see a transaction submitted by your node. After the transaction confirms, you will be able to withdraw your funds at the withdraw round that was specified in the prompt.
+
+If you change your mind and do no want to withdraw your funds, you can cancel your unlock request in `livepeer_cli` by selecting the following option:
+
+```
+14. Invoke "cancel unlock of broadcasting funds"
+```
+
+Upon selecting the option, you should be prompted to confirm that you would like to cancel your request to unlock your funds. After answering the wizard's prompt, you should see a transaction submitted by your node. After the transaction confirms, your unlock request will be cancelled. Another way to cancel an unlock request is to deposit more funds.
+
+If you want to withdraw your funds, in `livepeer_cli` select the following option:
+
+```
+15. Invoke "withdraw broadcasting funds"
+```
+
+If your unlock is not complete (i.e. your withdraw round is in the future), you will not be able to withdraw. Once your unlock is complete (i.e. your withdraw round is the current round or in the past), you should be prompted to confirm that you would like to withdraw your funds. After answering the wizard's prompt, you should see a transaction submitted by your node. After the transaction confirms, your withdrawal will be complete and can see your empty deposit and reserve by refreshing the wizard.
+
+### Configuring broadcasting preferences
+
+You can configure the following broadcasting preferences using the wizard:
+
+- The maximum price per pixel you are willing to pay. See the orchestrator section on [configuring payment parameters](#configuring-payment-parameters) for more details on how pricing per pixel works
+- The set of video profiles that you want your input video to be transcoded into
+
+In `livepeer_cli`, select the following option:
+
+```
+16. Set broadcast config
+```
+
+First, you will set the maximum transcoding price. You will be prompted for a maximum price relative to certain number of pixels. For example, you can set your maximum price to 0.1 wei per pixel if you set the maximum price to 1 wei for 10 pixels. The default number of pixels will be 1.
+
+Then, you will pick a set of video profiles that you would like your input video to be transcoded to. The set of video profiles presented in the wizard correspond to the [standard video profiles](https://github.com/livepeer/lpms/blob/master/ffmpeg/videoprofile.go) currently supported by the node. Future releases will offer greater flexibility around customizing video profiles to use for transcoding.
+
+### Broadcasting video
+
+See the [broadcasting guide](https://livepeer.readthedocs.io/en/latest/broadcasting.html) for information on sending video into your broadcaster node.
```

Comment:
I think there should be a "verify your transcoding setup" section at the top of the broadcasting section, and should give the minimum viable commands to stream then view a video transcoded by your node, in order to validate that your transcoding is working correctly. Can address in a future PR.

Reply:
There is already a section on this in the "Testing" sub-section under the "Transcoding" section. I updated the title of the sub-section to "Testing your setup" in f737780